resources/resincli.robot

```diff
@@ -145,12 +173,145 @@ Check enabling supervisor delta on "${application_name}"
     Process ${result}
-    ${result} =  Run Process    sed -i '6i echo \"${message}\"' start.sh    shell=yes    cwd=${directory}
+    ${result} =  Run Process    sed -ie 's/Hello World!/${message}/g' start.sh    shell=yes    cwd=${directory}
     Process ${result}
     ${result} =  Run Process    git add .    shell=yes    cwd=${directory}
     Process ${result}
-    ${result} =  Run Process    git commit -m "Console message added"    shell=yes    cwd=${directory}
+    ${result} =  Run Process    git commit -m "Console message added: ${message}"    shell=yes    cwd=${directory}
+    Process ${result}
+
+Add console command "${command}" to "${directory}"
+    ${result} =  Run Process    git config --global user.email "%{email}"    shell=yes    cwd=${directory}
+    Process ${result}
+    ${result} =  Run Process    sed -i '/while true/ i ${command}' start.sh    shell=yes    cwd=${directory}
+    Process ${result}
+    ${result} =  Run Process    sed -i '/while true/ i echo "Running command: ${command}"' start.sh    shell=yes    cwd=${directory}
+    Process ${result}
+    ${result} =  Run Process    git add .    shell=yes    cwd=${directory}
+    Process ${result}
+    ${result} =  Run Process    git commit -m "Console command added: ${command}"    shell=yes    cwd=${directory}
+    Process ${result}
+
+Check API endpoints of resin-supervisor
+    ${appId} =    Get "ID" from application "${application_name}"
+    ${deviceId} =    Get "ID" from device "${device_uuid}"
+    ${token} =    Get authentication token
+    Ping "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Blink "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Reboot "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Purge data of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Restart container's application of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Regenarate supervisor API key of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Update supervisor API key of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Stop container's application of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Start container's application of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Get application state of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+    Get device state of "${deviceId}" from application "${appId}" using "${token}" via the API proxy
+
+Get authentication token
+    ${result} =  Run Process    cat /root/.resin/token    shell=yes
+    Process ${result}
+    [Return]    ${result.stdout}
+
+Ping "${deviceId}" from application "${appId}" using "${token}" via the API proxy
```

Comment:
Can we use resin cli instead of curl urls? Since Resin CLI implicitly calls these URLS